packtools/sps/validation/article_citations.py

```diff
@@ -0,0 +1,223 @@
+from packtools.sps.models.article_citations import ArticleCitations
+from packtools.sps.validation.exceptions import ValidationArticleCitationsException
+
+
+class ArticleCitationsValidation:
+    def __init__(self, xmltree, publication_type_list=None):
+        self._xmltree = xmltree
+        self.article_citations = list(ArticleCitations(self._xmltree).article_citations)
```

Comment:
@Rossi-Luciano ArticleCitations(self._xmltree).article_citations é um gerador, mas qual é a necessidade de convertê-lo  a uma lista?

Reply:
@Rossi-Luciano cria uma **outra** classe chamada ArticleCitationValidation (singular), mantenha esta

Reply:
Para instanciar ArticleCitationValidation (singular), vc pode usar o item de `ArticleCitations(self._xmltree).article_citations`

Reply:
Feito isso, vc modifica `ArticleCitationsValidation (plural)` para ter um for e executar para cada element-citation a seq de validações de `ArticleCitationValidation (singular)`

Reply:
OK, @robertatakenaka